journals/apps/journals/models.py

```diff
@@ -366,6 +366,22 @@ def transcript(self):
             print('Exception trying to read transcript', err)
             return None
 
+    @property
+    def extended_view_url(self):
```

Comment:
Instead of adding `extended_view_url ` property here we should modify `gather_videos` command to update `view_url` at the time of video import.